CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.4)
 
-project(LibVNCServer C)
+project(LibVNCServer VERSION 0.9.12 LANGUAGES C)
```

Comment:
I see the point of using CMake-provided functionality, but this commit is missing a tiny bit:

> The VERSION option may not be used unless policy CMP0048 is set to NEW.

https://cmake.org/cmake/help/v3.4/command/project.html

Reply:
If you do a cmake_minimum_required(VERSION X) then all policies introduced in all prior versions are automatically set to NEW. CMP0048 was introduced in 3.0, so it is automatically set to NEW here anyway.

Reply:
:cherries: -picked